src/frontend/src/app/components/events/event-routing.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { NotfoundComponent } from '../../components/layout/notfound/notfound.component';
+import { EventAddComponent } from './event-add/event-add.component';
+import { EventListComponent } from './event-list/event-list.component';
+
+const eventRoutes: Routes = [
```

Comment:
You will want to add the layout component here to show the header.

This is my example from the profile component I am working on.


const routes: Routes = [

  { path: 'profile', redirectTo: 'profile/view', pathMatch: 'full' },
  {
    path: 'profile',
    component: LayoutComponent,
    children: [
      { path: 'view', component: ProfileComponent },
      { path: 'childadd', component: ProfileChildAddComponent },
    ],
  }
];

Reply:
I defaulted the `/events` path to `/home` and removed `/events/list` since that component just displays a single card at a time and I'm not sure we'll actually use `/events/list` in the future. I'm open to suggestions though!